Jenkinsfile.github

```diff
@@ -25,8 +25,7 @@
  */
 @Library('csm-shared-library') _
 
-def promotionToken = ~"(main|release\\/.*)"
-def isStable = env.TAG_NAME != null || env.BRANCH_NAME ==~ promotionToken ? true : false
+def isStable = env.TAG_NAME != null ? true : false
```

Comment:
Do we want *all* tags to create a stable build or only ones that match semver semantics?

Reply:
https://semver.org/#is-there-a-suggested-regular-expression-regex-to-check-a-semver-string

Would have to see if one of those works with groovy... or tweak it if necessary.  I think csm-shared-library is a better place to add it.